tests/plugins/uploadfile/uploadfile.js

```diff
@@ -226,5 +226,21 @@ bender.test( {
 		loader.abort();
 
 		assert.areSame( 'abort', loader.status, 'Loader status' );
+	},
+
+	// #2714
+	'test waitForImage when image is loaded': function() {
```

Comment:
This test should not be here, as it's connected to the test helper, not to the plugin itself. In fact helpers usually don't have tests.